adafruit_pyportal.py

```diff
@@ -41,6 +41,12 @@
   https://github.com/adafruit/circuitpython/releases
 
 * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
+"""
+"""
+04/19/2019
+John Turner
+Added ability to call the fetch object with a URL fetch(URL) this allows the URL to be updated once pyportal is initialized
```

Comment:
Please remove changenotes from the file, they'll be reflected in the release notes.

Reply:
Fixed